apps/api/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "type": "module",
   "scripts": {
-    "test": "NODE_ENV=test cross-env ava --no-worker-threads",
+    "test": "npm run prisma-generate && NODE_ENV=test cross-env ava --no-worker-threads",
```

Comment:
so here we are adding the `prisma-generate` command before we run the tests. This preloads the models we have defined into the database client getters. This ensures that all the models are available when running the tests, because otherwise the tests might complain saying "you are trying to mock getting this model, but i dont know what this model is". does this make any sense?